src/common/components/beta-notification/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import localforage from 'localforage';
+
+import { BETA_NOTIFICATION_TOGGLE } from '../../reducers/beta-notification';
```

Comment:
Action type imported directly from reducer. This is such a simple (and temporary) feature, that there is no need to add action creators boilerplate to it.